src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -386,6 +386,56 @@ protected override void MethodChecks(BindingDiagnosticBag diagnostics)
 
         protected sealed override void ExtensionMethodChecks(BindingDiagnosticBag diagnostics)
         {
+            if (ContainingType is { IsExtension: true, ExtensionParameter: { Type.IsStatic: false } extensionParameter } && !IsStatic &&
+                OperatorFacts.IsCompoundAssignmentOperatorName(Name))
+            {
+                if (extensionParameter.Name == "")
+                {
+                    diagnostics.Add(ErrorCode.ERR_InstanceMemberWithUnnamedExtensionsParameter, _location, Name);
```

Comment:
> Is this covered by test?

This is the line I was trying to point to (producing `ERR_InstanceMemberWithUnnamedExtensionsParameter` diagnostic)